source/CommonJobs/CommonJobs.Domain.EvalForm/EvaluationCalification.cs

```diff
@@ -40,13 +40,13 @@ public string Id
 
         public string EvaluatorEmployee { get; set; }
 
-        public string StrengthsComment { get; set; }
+        //public string StrengthsComment { get; set; }
```

Comment:
Could you remove the commented code or add another comment to clarify why it is commented?